app/javascript/src/LoginPage/RequestPasswordWrapper.jsx

```diff
@@ -0,0 +1,42 @@
+// @flow
```

Comment:
The name of the file should be simply `RequestPassword.jsx`.

Reply:
The other forms are called `LoginPageWrapper` or `SignupPageWrapper` and I didn't want to use an inconsistent name